### ISSUE TITLE
website update

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2561]
- [ ] list of projects
- [ ] logos of universities who have user projects
- [ ] killer features
- [ ] once the demo server for ARCE is up, update the how to find out about it section

**Comment by lbestock** [comment]
Ah that was vague and even inaccurate: we want logos of the UNIVERSITIES who are using Kiosk.

**Comment by luizaogs** [comment]
Ok, I added them to the website folder.

**Comment by lbestock** [comment]
Ideas from a discussion here.
People need to be able to find themselves (and quickly, without wading through a lot of text) on the website. As in, need to see that Kiosk is relevant to what they already do and to showing them it could be done better. We discussed a bit how to do this, thinking of pieces of archaeological endeavors, leading to different sections. What do archaeologists do? Record contexts, object photography, registration, inventory, epigraphy, pedestrian survey, sketch plans, produce government reports, publish (ha ha ha), coordinate research after fieldwork. No particular order. Right now it looks like @urapadmin might take more of the website brainstorming lead (others to help write copy) and @lbestock more of the graphic lead in getting ready for ARCE.

**Comment by urapadmin** [comment]
I'll take this on. It is a good thing to ponder over on the various trains I'll be on this week.

**Comment by urapadmin** [comment]
see #2093

**Comment by lbestock** [comment]
Alas I think we need to reconsider the banner image on the website. It takes up so much room without being so clearly about iPad recording in the field; having to scroll past it isn't ideal.

Website can be a jointly worked on project on the train? But it does strike me as one of the most important things to have done soon, though perhaps less critical before ARCE and more in the week afterwards when, using a little pot of soothing mint balm, someone remembers to check it out with more leisure.

**Comment by urapadmin** [comment]
Our landing page is a pretty standard way of designing a landing page. It is even the point of it that you must scroll down (which you will have to do anyhow). That has advantages, like having something aesthetically pleasing to be greeted with (instead of huge text blocks and overwhelming headlines everywhere). By making sure that the thing this is about is just at the bottom of the screen (and you can only do that with a landing images like this) you indicate that there is stuff to scroll down to. Works on mobile (not ours, really), desktop etc. Many things to take into consideration that are not immediately obvious. 
Because of these advantages that sort of landing page became so popular in the first place. I can still live with it. At least I have no idea at hand how to actually improve it. 

But we should of course try to place the logo there etc. Maybe then it is a bit more about field recording.

**Comment by luizaogs** [comment]
Any chance you want to add the list of projects to the website for ASOR (and I guess the university logos)? It's basically ready in the drive, I would just need to know in advance to add the new ones.

**Comment by urapadmin** [comment]
ja, actually I do want to work a bit on tutorials and the website this week before I start the next big project

**Comment by luizaogs** [comment]
Ok, it's all now there except for AAC. Who directs that? I can't find anything online, other than that it's in Turkey.

The list is in the website folder, along with the university logos.

**Comment by urapadmin** [comment]
Thank you. That made it really easy for me to integrate them.

**Comment by urapadmin** [comment]
new logo: Needs to be put on web site
